app/assets/stylesheets/osem.css.scss

```diff
@@ -86,3 +86,51 @@ p.comment-body {
 #proposal-info div dt, #proposal-info div dd {
   display: inline-block;
 }
+
+.room-name{
```

Comment:
Because that is for the admin schedule and I thought that osem-schedule.css was only for the public schedule. Do I move it?